src/main/java/school/hei/haapi/service/PaymentService.java

```diff
@@ -88,8 +89,28 @@ List<Payment> getByStudentIdAndFeeId(String studentId, String feeId) {
 
   public void computeRemainingAmount(String feeId, int amount) {
     Fee associatedFee = feeService.getById(feeId);
+    User student = associatedFee.getStudent();
+    User.Status[] statuses = new User.Status[2];
```

Comment:
i think studentStatusBetweenPayingFee is better to mark that it's an array of state of student status